packages/components/input/stories/input.stories.tsx

```diff
@@ -90,7 +90,12 @@ const PasswordTemplate: ComponentStory<typeof Input> = (args) => {
       <Input
         {...args}
         endContent={
-          <button className="focus:outline-none" type="button" onClick={togglePasswordVisibility}>
+          <button
+            aria-label="show password"
```

Comment:
Would be nice to change the `aria-label` according to the `isPasswordVisible` state.

```js
aria-label={isPasswordVisible ? "hide password : "show password"}
```

Reply:
You can try it out, and you will see that it does not get read out if you change a label dynamically like this.

Even if it would work, `aria-pressed` is the right pattern for toggle buttons like this.

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-pressed